src/main/kotlin/de/fuballer/mcendgame/main/component/entity/custom/entities/training_dummy/TrainingDummyEntity.kt

```
@@ -105,7 +106,9 @@ class TrainingDummyEntity(
 
     override fun doPush(entity: Entity) {}
 
-    override fun knockback(strength: Double, x: Double, z: Double) {}
+    override fun knockback(power: Double, xd: Double, zd: Double, source: DamageSource, damage: Float) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)